src/main/java/team4/footwithme/chat/service/ChatroomServiceImpl.java

```diff
@@ -105,6 +108,15 @@ public ChatroomResponse updateChatroom(Long chatroomId, ChatroomServiceRequest r
         return new ChatroomResponse(chatroom);
     }
 
+    /**
+     * 내가 참여한 채팅방 조회
+     */
+    @Transactional
```

Comment:
```suggestion
    @Transactional(readOnly = true)
```
데이터 읽기만 한다면 트랜잭션 격리 수준은 읽기 전용이 좋을 것 같아요

Reply:
넵 피드백 감사합니다!